src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -67,75 +63,64 @@ public BtToRequestSocket(BluetoothSocket socket, BtSocketDisconnectedCallBack ha
         mmOutStream = tmpOut;
     }
 
-    public void setPort(int port) {
-        mHTTPPort = port;
-    }
-
     public void run() {
+
         try {
             print_debug("start Socket with port: " + mHTTPPort);
+            mHTTPPort = 0; // just to make sure its zero
```

Comment:
:+1: 

Yup, the comment on run was my bad. I was thinking of start which we use in other places. Sorry.